Initium-Odp/src/com/universeprojects/miniup/server/scripting/wrappers/Item.java

```diff
@@ -113,6 +113,32 @@ public boolean adjustDurability(Long addDura)
 		return newDura < 0;
 	}
 	
+	public Long getDexPen() {
+		return (Long)this.getProperty("dexterityPenalty");
+	}
+	public Long getStrMod() {
+		return (Long)this.getProperty("strengthModifier");
+	}
+	public Long getIntMod() {
+		return (Long)this.getProperty("intelligenceModifier");
+	}
+	public Long getDamageReduction() {
+		return (Long)this.getProperty("damageReduction");
+	}
+	
+	public void setDexPen(Long newDexPen) {
+		this.setProperty("dexterityPenalty", newDexPen);
+	}
+	public void setStrMod(Long newStrMod) {
+		this.setProperty("strengthModifier", newStrMod);
+	}
+	public void setIntMod(Long newIntMod) {
+		this.setProperty("intelligenceModifier", newIntMod);
+	}
+	public void setDamageReduction(Long damageReduction) {
+		this.setProperty("damageReduction", damageReduction);
+	}
+	

```

Comment:
I'm not certain this is something we want to allow. Rogue dev can basically make OP items via scripting, with no log of the changes. Why we've always limited scripting to very few devs, since it has to be someone that can be trusted.